src/uu/tail/src/platform/windows.rs

```diff
@@ -57,3 +57,13 @@ impl Drop for ProcessChecker {
 pub fn supports_pid_checks(_pid: self::Pid) -> bool {
     true
 }
+
+#[inline]
+pub fn links_count(f: &File) -> u64 {
```

Comment:
please fix that :)

Reply:
I am not sure what is to be fixed here? We are going to need the `links_count` function to be able to detect if an open File object got deleted while we're reading it. I moved it to `platform/` because it's a platform dependent function.